app/src/main/java/ch/epfl/sdp/contamination/InfectionActivity.java

```diff
@@ -16,18 +16,58 @@
 
 import java.util.Date;
 
+import ch.epfl.sdp.AuthenticationManager;
 import ch.epfl.sdp.R;
 import ch.epfl.sdp.fragment.AccountFragment;
 import ch.epfl.sdp.location.LocationService;
 
+import static ch.epfl.sdp.AuthenticationManager.getActivity;
+
 public class InfectionActivity extends AppCompatActivity {
 
+
```

Comment:
Did you merge my updated version of this file?

Reply:
Oh ! I knew there was something odd. Yeah I'll merge your version